### PR TITLE
node: improve test coverage on proposal block

### PR DESCRIPTION
**Pull request by cmwaters**
## Description

Closes: #3156

### CODE REVIEW

File: node/node_test.go [hunk 1]
```diff
@@ -375,6 +378,120 @@ func TestMaxProposalBlockSize(t *testing.T) {
...
+	proxyApp := proxy.NewAppConns(cc)
+	err := proxyApp.Start()
+	require.Nil(t, err)
+	defer proxyApp.Stop() //nolint:errcheck // ignore for tests
+
+	logger := log.TestingLogger()
+
+	state, stateDB, _ := state(types.MaxVotesCount, int64(1))
+	stateStore := sm.NewStore(stateDB)
+	var maxBytes int64 = 1024 * 1024 * 2
```
Comment (melekes) [inline node/node_test.go:394]:
```suggestion
	const maxBytes int64 = 1024 * 1024 * 2
```

**Review by melekes — APPROVED** [review]
👍

File: node/node_test.go [hunk 2]
```diff
@@ -375,6 +378,120 @@ func TestMaxProposalBlockSize(t *testing.T) {
...
+	state.ConsensusParams.Block.MaxBytes = maxBytes
+	proposerAddr, _ := state.Validators.GetByIndex(0)
+
+	// Make Mempool
+	memplMetrics := mempl.PrometheusMetrics("node_test_2")
+	mempool := mempl.NewCListMempool(
+		config.Mempool,
+		proxyApp.Mempool(),
+		state.LastBlockHeight,
+		mempl.WithMetrics(memplMetrics),
```
Comment (marbar3778) [inline node/node_test.go:404]:
```suggestion
		mempl.WithMetrics(mempl.NopMetrics()),
```
to avoid the panic in ci

Comment (cmwaters) [reply node/node_test.go:404]: I've made the change for the other tests as well

**Pull request merged**